include/hip/hcc_detail/hip_runtime.h

```diff
@@ -279,6 +279,8 @@ extern hipStream_t ihipPreLaunchKernel(hipStream_t stream, size_t grid, size_t b
                                        grid_launch_parm* lp, const char* kernelNameStr);
 extern void ihipPostLaunchKernel(const char* kernelName, hipStream_t stream, grid_launch_parm& lp);
 
+extern hipError_t ihipGetDeviceProperties(hipDeviceProp_t* props, int device);
```

Comment:
Why do we need this here? This should go to src/hip_module.cpp or src/hip_hcc_internal.h.

Reply:
Fixed by moving extern declaration of ihipGetDeviceProperties to hip_module.cpp.